src/main/java/com/plaid/client/response/depositswitch/DepositSwitchGetResponse.java

```diff
@@ -0,0 +1,51 @@
+package com.plaid.client.response.depositswitch;
+
+import com.plaid.client.response.BaseResponse;
+
+public final class DepositSwitchGetResponse extends BaseResponse {
+  private String depositSwitchId;
+  private String targetItemId;
+  private String targetAccountId;
+  private String state;
+  private String dateCreated;
+  private Boolean isAllocatedRemainder;
+  private Boolean accountHasMultipleAllocations;
+  private Float percentAllocated;
```

Comment:
https://github.com/plaid/plaid-go/pull/96#discussion_r371554800

Reply:
⚡️

Reply:
⚡️ 

> Looks good for the most part - is there any way to be less redundant with the tests we've written? It looks like the code for `testDepositSwitchGetSuccess` is a superset of all other tests.

Yeah, there is a lot of redundancy. I chose to do it this way because I wasn't sure if it is a java convention to test routes like this. further, there seems to be no deviation from the format of testing every route throughout the entire testing suite